src/components/FindWallet/WalletPersonasSidebar.tsx

```diff
@@ -499,16 +356,53 @@ const WalletPersonasSidebar = ({
   ]
 
   return (
-    <Container>
-      <ProfileFilterPrompt>
+    <Flex direction="column" gap={4} p={{ base: 4, sm: 0 }}>
+      <Box
+        as="span"
+        m={0}
+        fontWeight="normal"
+        fontSize="sm"
+        p="0 1.2rem"
+        lineHeight="1.3rem"
+        textAlign="center"
+        color="secondary"
+      >
         Choose the profile that matches your type of user and filter the wallet
         list
-      </ProfileFilterPrompt>
+      </Box>
       {personas.map((persona, idx) => {
         return (
-          <Persona
-            isDark={theme.isDark}
-            selected={selectedPersona === idx}
+          <Flex
+            direction="column"
+            alignItems="flex-start"
+            padding={6}
+            background={
+              selectedPersona === idx
+                ? isDark
+                  ? "primary900"
+                  : "primary200"
+                : isDark
+                ? "black400"
+                : "primary100"
+            }
+            borderRadius={1}
+            cursor="pointer"
+            transition="0.5s all"
+            sx={{
+              h3: {
+                color: "text",
+              },
```

Comment:
Add this color directly on the `<Heading as="h3"` component instead of using the `sx` prop.